vignettes/pander.Rmd

```diff
@@ -0,0 +1,197 @@
+---
+title: "Randering markdown with pander"
+author: "Roman Tsegelskyi, Gergely Daróczi"
+date: "`r Sys.Date()`"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{Pandoc.table}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r, echo = FALSE, message = FALSE}
+knitr::opts_chunk$set(collapse = T, comment = "#>")
+library(pander)
+library(tables)
+panderOptions('knitr.auto.asis', FALSE)
+panderOptions('plain.ascii', TRUE)
+```
+
+`Pander` is designed to provide a minimal and easy tool for rendering `R` objects into [Pandoc](http://pandoc.org/)'s markdown. This vignette aims to introduce `pander` package and it's core pieces of functionality. It is intented to be a general overview with pointers to places with detailed information. This vignette will talk about:
+
+* core functionality for rendering objects in `Pandoc`'s markdown with generic S3 [pander method](#rendering-r-objects).
+* functionality for capturing various information when evaluating R expressions with [evals](#evals).
+* report generation with [Pandoc.brew](#brew-to-pandoc).
+* globally adjustable options through [panderOptions/evalsOptions](#general-options).
+
+## Rendering R objects
+
+The core functionality of `pander` is centered around rendering `R` objects into `Pandoc`'s markdown. Let's dive in the demo of the most common usage of `pander`:
+
+```{r}
+pander(head(iris))
+pander(head(mtcars[1:5]))
+pander(tabular( (Species + 1) ~ (n=1) + Format(digits=2)*
+         (Sepal.Length + Sepal.Width)*(mean + sd), data=iris ))
+```
+
+As you have probably guesse, this is achieved via generic `pander` `S3` method. Out of the box, `pander` supports a variety of classes:
+
+```{r}
+methods(pander)
+```
+
+If you think that pander lacks support for any other R class(es), please feel free to leave change request [here](https://github.com/Rapporter/pander/issues) and we will be happy to extend the package.
+
+Under the hood, `pander` S3 methods rely on different `pandoc.*` methods, where most of functionality is implemented in `pandoc.table` which is used for rendering tables. `pandoc.table` provides  a truly rich functionality with many different options that `pander` inherits. For more usage/implementation details and examples, please refer to specialized vignette, which can be accessed by `vignette('pandoc_table')` or available online [here]().
```

Comment:
Can we mention that it's similar to `knitr::kable` with markdown only although with some extra features?